crates/muvm/src/guest/bin/muvm-guest.rs

```diff
@@ -50,6 +50,12 @@ fn main() -> Result<()> {
             .spawn()?;
     }
 
+    // Before switching to the user, start another instance of muvm-server to serve
+    // launch requests as root.
+    if let Some(muvm_server_path) = find_in_path("muvm-server")? {
+        Command::new(muvm_server_path).spawn()?;
+    }
```

Comment:
Are there situations where `muvm-server` is expected to be missing? If not this could just be:

```suggestion
    Command::new("muvm-server").spawn().context("Failed to find `muvm-server`")?;
```

Reply:
Applied, thanks.